### ISSUE TITLE
[FEATURE REQUEST] - Please add supplementary prompt word function to SD Prompt Reader node..

### ISSUE THREAD

**Issue by cheezmil** [receyuki/comfyui-prompt-reader-node#19]
### Summary

![QQ浏览器截图20231108213609](https://github.com/receyuki/comfyui-prompt-reader-node/assets/134115886/f206cd91-9db2-4d99-a72f-7a06a17e0ec4)
Can add a button on this node and then modify the content in the text box? I think it would be much more convenient....Thank you very much

### Basic Example

1. Add a button in the SD Prompt Reader node

2. Click to modify the content in the text box

**Comment by receyuki** [comment]
其他人可能也有类似的问题所以我就不用中文了
tl;dr: Due to the limitations of ComfyUI, this feature cannot be implemented. But you can achieve similar functions by adding another node.

After a simple explanation of the working principle of this node, you might be able to understand why
1. When this node runs, it reads the selected image
2. It reads and parses metadata from the image.
3. Outputs the prompt and parameters.
4. Displays the prompt and parameters in the text box below.

In other words, the text box below is only for display at the final stage, and you may have noticed that the text box can only be updated after the node runs. Therefore, editing the content in the text box is pointless.

But if you just want to simply add content, you can consider using the `Prompt Merger` node to merge the existing prompt with another string.
